### DISCUSSION TITLE
Cant find the zksync plugin on the plugin tab in remix

### DISCUSSION THREAD

**Discussion by Blockchainwiz001** [Cyfrin/foundry-full-course-cu#2211] [General]
Please can someone help me here? I'm following the tutorial video, but on the plugin tab where Mr Patrick clicked, when i clicked it and searched for zksync, it was blank

**Comment by EngrPips** [comment]
Hello sir @PatrickAlphaC it seem like people are having issues adding `ZkSync` to remix

**Comment by cromewar** [comment]
Hello, @Blockchainwiz001; that might be a problem with Remix at that moment. I just checked, and it is loading properly; Do you still have the same issue?

**Comment by Vodvalex** [comment]
It takes a long time to deploy and verify to the zk sync. Definitely not the brief delay. Has anyone got the same issue?

**Reply by EngrPips** [reply]
Everything works seamlessly for me.

**Comment by cromewar** [comment]
Closing this issue due to lack of follow up from the creator.

**Comment by TheWeb3geek** [comment]
I still cant find the zksync plugin, tried different ways, had to change browser and network provider but still got nothing.......anyone has a solution?

**Reply by EngrPips** [reply]
You can move on deploying to the `zksync` network is optional
